src/app/pages/Languages.svelte

```diff
@@ -23,8 +28,21 @@
     sources: [],
   };
 
-  onMount(async () => {
+  async function getLanguageSources() {
     languageInformation.sources = await worker.getFilesFromStorage();
+  }
+
+  // listen to changes to the package compilation and recalculate word count
+  compileSuccess.subscribe((currentValue) => {
+    downloadReady = currentValue;
+    if (currentValue) {
+      getLanguageSources();
```

Comment:
Why does `getLanguageSources()` get called here? As far as I understand, it should only be called `onMount()` to the list of possible languages ONCE.

Reply:
This was to ensure that the word count was up to date when re-compilation occurred but perhaps it is better to only update the word count when a dictionary is added. I can pass `getLanguageSources()` down to the source components instead.